lib/resources/r4/charge_item_create.yaml

```diff
@@ -291,7 +291,7 @@ fields:
       <ul>
         <li>Custom-attribute-name and Custom-attribute-value are both required</li>
         <li>id and valueString for custom-attribute-value are required</li>
-        <li>Custom-attribute-value can be of type integer, string or decimal</li>
```

Comment:
nit: let's update 'Custom-attribute-value' and 'Custom-attribute-name' to be lower case like we do for modify and other field names.